clients/common/cblas_interface.cpp

```diff
@@ -572,13 +572,14 @@ double abs_helper(T val)
 template <>
 double abs_helper(hipblasComplex val)
 {
-    return std::abs(val.x) + std::abs(val.y);
+    return std::abs(val.real()) + std::abs(val.imag());
+    ;
```

Comment:
Good catch Daine. This is what caused CI to fail as well.

Reply:
> 
> 
> Looks like a typo here.

No typo.  An extra semicolon is just a harmless empty statement.

hip-clang works fine in my build environment. I'll investigate why the CI is failing.

Reply:
Also, maybe this PR should be considered draft; I cannot attest to it passing all tests yet. It was submitted early because others submitted an incomplete fix while I was working on this fix.